echo/frontend/src/components/report/CreateReportForm.tsx

```
@@ -85,112 +84,106 @@ export const CreateReportForm = ({ onSuccess }: { onSuccess: () => void }) => {
+                          {conversationCounts.pending}{" "}
+                          {conversationCounts.pending === 1
+                            ? t`conversation`
+                            : t`conversations`}{" "}
+                          <Trans>processing</Trans>
+                        </Text>
+                      </Group>
+                      <Text size="sm" c="orange.6">
+                        ~30 <Trans>min</Trans>
+                      </Text>
+                    </Flex>
+                  </>
```

Biome (1.9.4)

[error] Avoid using unnecessary Fragment.

A fragment is redundant if it contains only one child, or if it is the child of a html element, and is not a keyed fragment.
Unsafe fix: Remove the Fragment


(lint/complexity/noUselessFragments)